https/callBuilder.go

```diff
@@ -683,12 +684,12 @@ func (cb *defaultCallBuilder) CallAsText() (string, *http.Response, error) {
 			return "", result.Response, fmt.Errorf("response body empty")
 		}
 
-		body, err := io.ReadAll(result.Response.Body)
+		bodyBytes, err := io.ReadAll(result.Response.Body)
 		if err != nil {
 			return "", result.Response, fmt.Errorf("error reading Response body: %v", err.Error())
 		}
-
-		return string(body), result.Response, err
+		result.Response.Body = io.NopCloser(bytes.NewBuffer(bodyBytes))
```

Comment:
Why is this being set?

Reply:
Because when you read the data this reader object reset itself.